lib/ansible/modules/packaging/os/pkgng.py

```diff
@@ -33,7 +33,7 @@
     state:
         description:
             - State of the package.
```

Comment:
make note that latest is available since 2.7

Reply:
I realize I may have misunderstood your original request. Did you want me to add `version_added: 2.7` to the `state` option, or just a note in the `state` comments section indicating that `latest` is an option as of 2.7?

Reply:
comment, state option is preexisting, but the new choice is not

Reply:
Ah got it, sorry about that.

Just removed the `version_added:` with c5bb88e.

Lemme know if you'd like any other changes.